docs/src/app/components/pages/components/popover.jsx

```diff
@@ -87,6 +87,16 @@ let PopoverPage = React.createClass({
       <ComponentDoc
         name="Popover"
         componentInfo={componentInfo}>
+        
+        <Paper style = {{marginBottom: '22px'}}>
+          <CodeBlock>
+          {
+            '//Import statement:\nimport Popover from \'material-ui/lib/popover\';\n\n' +
```

Comment:
CI is posting a linter issue from an unrelated area ( `events.js` which doesn't even seem to be part of this repository... likely npm/gulp related, using bleeding edge or old versions )

Agreed, the documentation does need some heavy duplication removal, and updating to be current with new `React` event binding behaviors... because *all* the documentation example cost posts `React` warnings.